sevenseconds/config/__init__.py

```diff
@@ -39,28 +40,58 @@
         'auth'              # OAuthServices Object (exp. for Account List and AWS Credentials Service)
     ))
 
+SharedData = namedtuple(
```

Comment:
Why did you use SharedData? Lastest AMI isn't the same for all accounts.

Reply:
See the comment above. In the new model it is, because with the staging channel we don't need the manual sharing step anymore.